### PR TITLE
Cargo Techs now have the same power as the Quartermaster to approve crates at the supply console

### PR DESCRIPTION
**Pull request by Sirryan2002**
## What Does This PR Do
Cargo Techs now have the same power as the Quartermaster to approve crates at the supply console.

## Why It's Good For The Game
With the economy changes in, purchases at the cargo console have to go through a more in-depth approval process. One issue that has arisen out of this is the requirement the QM have sole approval power over crates, this change seems to be rather controversial and many players have expressed a dislike of it.

I think this is because being a CT is a rather dry job especially when you can no longer just order fun crates and play around with them which was a big perk of being a CT before the econ rework. Additionally, the cargo account tends to get rather full of space cash by round end because it largely goes unused except for covering other department's purchases so giving CTs a chance to drain the account seems like something worth trying.

As I've covered in a previous discussion post #19734, there's some crates that departments just absolutely need, an absent QM shouldn't get in the way of departments trying to do stuff so it will be good for CTs to be able to step in.
## Testing
Compiled, ran, tested approval as CT

## Changelog
:cl:
tweak: Cargo Techs now have the same power as the Quartermaster to approve crates at the supply console.
/:cl:

### CODE REVIEW

**Comment by Gatchapod** [comment]
While I understand why people find this frustrating, I believe this change would be a net negative.

People still try to make purchases using supply account and CTs, who are often played by less experienced players, will probably hit that approve button without much consideration. There are no permanent logs of who approved an order, so QM cannot even hold the person accountable.

If cargo is getting halted by a lack of QM, command should get off their butts and address the issue.

**Comment by Sirryan2002** [comment]
> While I understand why people find this frustrating, I believe this change would be a net negative.
> 
> People still try to make purchases using supply account and CTs, who are often played by less experienced players, will probably hit that approve button without much consideration. There are no permanent logs of who approved an order, so QM cannot even hold the person accountable.
> 
> If cargo is getting halted by a lack of QM, command should get off their butts and address the issue.

The account getting drained by negligent purchases is less of an issue since CT's can't order crates with department accounts besides the supply account. There is logging in the form of printed order forms.

The issue with command "solving" the issue of an absent QM is that a non-existent command team can also accompany an absent QM. Furthermore, if this is a round-start issue, it can significantly delay station goal, research, and other crates. 

I maintain that it was a mistake to access lock a core department mechanic to only the "head" of a department.

**Comment by Sirryan2002** [comment]
I may tweak this to be a bit different when I get the time this week.

**Comment by Sirryan2002** [comment]
will deconflict this one once the economy manager is merged

**Review by S34NW — APPROVED** [review]
A shame, but the code looks good

File: code/modules/supply/supply_console.dm [hunk 1]
```diff
@@ -39,7 +39,7 @@
 	ui_interact(user)
 
 /obj/machinery/computer/supplycomp/proc/has_qm_access(list/access)
-	return (ACCESS_QM in access) ? TRUE : FALSE
+	return (ACCESS_CARGO in access) ? TRUE : FALSE
```
Comment (hal9000PR) [inline code/modules/supply/supply_console.dm:42]: I feel like this proc should be renamed

**Review by hal9000PR — APPROVED** [review]

**Pull request merged**